src/assets/text/authors.js

```diff
@@ -0,0 +1,37 @@
+export default {
+    // do not delete section. delete individuals as needed. modify content as needed
+    // currently only 'fullName', 'firstName', 'initials', 'profile_link', and 'contribution' are used
+    primaryAuthors: [
+      {
+        firstName: 'Althea',
+        lastName: 'Archer',
+        fullName: 'Althea Archer',
+        initials: 'AA1',
```

Comment:
```suggestion
        initials: 'AAA',
```

Reply:
thanks for catching those holdovers from the template!